src/explore-education-statistics-frontend/src/modules/find-statistics/components/PublicationSectionBlocks.tsx

```diff
@@ -16,6 +16,9 @@ import {
 import React from 'react';
 import VisuallyHidden from '@common/components/VisuallyHidden';
 
+export const permittedEmbedDomains =
```

Comment:
@ntsim this all works locally, but I wasn't sure if this also needed to be added to next.config.js in either its existing `publicRuntimeConfig` config block or a new `serverRuntimeConfig` config block?

Reply:
Yeah, in order to expose these environment variables for use client-side usage you'll need to add it to `publicRuntimeConfig`. I believe this is just a convention of how we write environment variables but typically you'd prefix Next.js environment variables with `NEXT_` to expose them for client-side use. I'm wondering if this`checkOrigin` prop is even necessary since we restrict what the admin users can set and the public frontend CSP will refuse to load anything that's not allowed

Reply:
Nice one, sounds good thanks!  I'll see if I can remove the prop entirely from the frontend.

Reply:
Removed from EmbedBlock entirely - relying on validation in Admin and the front-end's CSP policy to only allow permitted domains to be viewed in the embed block's iframe.